npm-shrinkwrap.json

```diff
@@ -469,11 +472,380 @@
       "version": "0.3.0"
     },
     "fs-extra": {
-      "version": "0.26.7"
+      "version": "1.0.0"
     },
     "fs.realpath": {
       "version": "1.0.0"
     },
+    "fsevents": {
```

Comment:
Can you remove this (everything under fsevents) - this will cause an error in our internal CI.